ucm2/USB-Audio/Realtek-ALC1220-VB-Desktop-HiFi.conf

```diff
@@ -0,0 +1,60 @@
+SectionDevice."Speaker" {
+        Comment "Speakers"
+        Value {
+               PlaybackChannels "8"
+               PlaybackPCM "hw:ALC1220VBDT,0"
+               JackControl "Speaker Jack"
+               PlaybackMixerElem "Speaker"
+        }
+}
+
+SectionDevice."Headphone" {
+        Comment "Front Headphone"
+        Value {
+                PlaybackChannels "2"
```

Comment:
Remove 2 channel lines. It's the default now.

Reply:
Done